rpm-arm64/Dockerfile

```diff
@@ -103,6 +115,9 @@ RUN /bin/bash -l -c "rvm install 2.7 --with-openssl-dir=${CONDA_PATH} && rvm cle
 RUN /bin/bash -l -c "gem install bundler --version $BUNDLER_VERSION --no-document"
 RUN echo 'source /usr/local/rvm/scripts/rvm' >> /root/.bashrc
 
+# Override updated linker system-wide
+RUN ln -sf /usr/local/binutils/bin/ld /usr/bin/ld
```

Comment:
Won't this create problem if we end up using another binutils tool from the distribution?

Reply:
I'll be honest: I don't know yet. It is what's done in [rpm-x64](https://github.com/DataDog/datadog-agent-buildimages/blob/2333b0b25faf155b31e4e054aad5677c6ab62008/rpm-x64/Dockerfile#L131-L142) and [suse-x64](https://github.com/DataDog/datadog-agent-buildimages/blob/2333b0b25faf155b31e4e054aad5677c6ab62008/suse-x64/Dockerfile).

I'll see if I can use it only on the problematic part instead of this `ln -sf`

Reply:
Oh if we've been doing so for at least a year I suppose it's fine enough for the time being